### PR TITLE
Feat/pyinstaller ci

### PR DESCRIPTION
**Pull request by 0xArdi**
Adds support for building pearl binary in CI.

### CODE REVIEW

**Review by truemiller — APPROVED** [review]

File: templates/trader.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ description: "A single-agent service (sovereign agent) placing bets on Omen"
 hash: bafybeihrogxx2kgl7euvhqsbh33fi4tzuupxodoheraqxd6lkvtvljewtu
 image: https://operate.olas.network/_next/image?url=%2Fimages%2Fprediction-agent.png&w=3840&q=75
 configuration:
+  trader_version: v0.16.4
```
Comment (0xArdi) [inline templates/trader.yaml:6]: This needs to be updated when a new version of trader is meant to be used.

Comment (jmoreira-valory) [reply templates/trader.yaml:6]: We reference it by the hash (which is what is fetched ultimately). What impact does this version have?

Comment (jmoreira-valory) [reply templates/trader.yaml:6]: In particular, we have a recent hash update #225 , but there is no release of the repository. What happens in this situation?

Comment (0xArdi) [reply templates/trader.yaml:6]: @jmoreira-valory this specifies the aea binary that needs to be used when running the agent. It may be that deps change.

**Review by solarw — APPROVED** [review]
lgtm

**Comment by jmoreira-valory** [comment]
@0xArdi please make sure trader.yaml is consistent with the .ts file as well, as discussed.

**Comment by 0xArdi** [comment]
@jmoreira-valory its addressed

**Review by jmoreira-valory — APPROVED** [review]

**Pull request merged**